src/tests/lib/libffi/libffi_test.c

```diff
@@ -0,0 +1,85 @@
+#include <stdio.h>
+#include <ffi.h>
+#include <embox/test.h>
+
+EMBOX_TEST_SUITE("lib/ffi_test");
+
+int expected;
+
+int test_function(int actual) {
+	test_assert_equal(actual, expected);
+	return actual;
+}
+
+TEST_CASE("test call simple function") {
+	ffi_cif cif;
+	ffi_type *args[1];
+	void *values[1];
+	ffi_arg rc;
+	
+	int actual;
+	args[0] = &ffi_type_sint;
+	values[0] = &actual;
+
+	if (ffi_prep_cif(&cif, FFI_DEFAULT_ABI, 1, &ffi_type_sint, args) == FFI_OK) {
+		expected = 1;
+		actual = expected;
+		ffi_call(&cif, test_function, &rc, values);
```

Comment:
As I see here[ https://linux.die.net/man/3/ffi_call]( https://linux.die.net/man/3/ffi_call) ffi_call has following signuture 
```void ffi_call(ffi_cif *cif, void (*fn)(void), void *rvalue, void **avalue);```

but you use `ffi_call(&cif, test_function, &rc, values);` although it must be `ffi_call(&cif, FFI_FN(test_function), &rc, values);`